lldb/test/API/lang/swift/macCatalyst/TestSwiftMacCatalyst.py

```diff
@@ -29,7 +29,7 @@ def test_macCatalyst(self):
         lldbutil.run_to_source_breakpoint(self, "break here",
                                           lldb.SBFileSpec('main.swift'))
         self.expect("image list -t -b",
-                    patterns=["x86_64-apple-ios13.0.0-macabi a\.out",
+                    patterns=[arch + ".*-apple-ios.*-macabi a\.out",
                               "x86_64.*-apple-ios.*-macabi Foundation",
```

Comment:
Why don't we need to change this line? It also has x86 hardcoded.

Reply:
Good question ;) I just cherry picked your commit from our internal branch. I'll change that too